content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm.md

```diff
@@ -75,11 +104,45 @@ For a Docker Linux container:
      gcr.io/datadoghq/agent:7
    ```
    Replace `<YOUR_DD_API_KEY>` with your [Datadog API][5].
-
+   <div class="alert alert-info">
+      You can also optionally configure the following:
+      <ul>
+         <li><a href="#lib-docker">Specifying tracing library versions.</a></li>
+         <li><a href="#env-docker">Tagging observability data by environment.</a></li>
+      </ul>
+   </div>
 3. Restart the Docker containers.
 4. [Explore the performance observability of your services in Datadog][6].
 
-   <div class="alert alert-info">You can optionally set an environment for your services and other telemetry that pass through the Agent. Read <a href="#env-docker">tagging observability data by environment</a> to learn how. </div>
+### Specifying tracing library versions {#lib-docker}
```

Comment:
These instructions are incorrect. Apologies, I didn't specify this in my initial communication with you.

To specify tracing libraries and versions for docker, set `DD_APM_INSTRUMENTATION_LIBRARIES` when running the  `bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script_docker_injection.sh)"` installation script in the step labeled "Install the library injector"

```
DD_APM_INSTRUMENTATION_LIBRARIES="java:1.25.0,python" bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script_docker_injection.sh)"
```

The rest of the description is correct.

Reply:
@jonbodner - Thanks for the keen eye! I updated this section to correctly set the library versions inside the installation command. Let me know how this looks.